rest-api/src/main/java/gov/cms/qpp/conversion/api/model/Metadata.java

```diff
@@ -293,8 +304,8 @@ public void setValidationStatus(Boolean validationStatus) {
 	 * @return True for a CPC+ conversion, false otherwise.
 	 */
```

Comment:
javadocs should be updated to reflect the new purpose for this attribute

Reply:
Good catch!